c2cgeoportal/__init__.py

```diff
@@ -266,6 +266,9 @@ def new_f(root, request):
         renderer=renderer
     )
 
+    config.add_static_view('node_modules', config.get_settings().get('node_modules_path'))
+    config.add_static_view('closure', config.get_settings().get('closure_library_path'))
```

Comment:
For non-ngeo projects, `get_settings().get('node_modules_path')` will return `None`. Did you make sure that this will not make `add_static_view` fail?